src/basicfuns.jl

```diff
@@ -353,6 +353,11 @@ function _log1pmx_ker(x::T) where T <: Union{Float32, Float64}
              0.11201972567415432,
              0.143418239946679)
     end
+    # Mathematically equivalent to:
```

Comment:
IMO if one wants to be precise, it's not "mathematically equivalent" but rather an approximation.